client/my-sites/plugins/hooks/types.ts

```diff
@@ -16,6 +16,11 @@ export type Site = {
 	ID: number;
 	title: string;
 	canUpdateFiles?: boolean;
+	is_a4a_client?: boolean;
+	capabilities?: {
+		activate_plugins: boolean;
+		update_plugins: boolean;
+	};
```

Comment:
Do we need to mix snake_case with camelCase here?

Reply:
Yeah, that's how the data is currently formatted in the app. I think it is a combination of API and defined values. 

I removed this in favour of existing types: https://github.com/Automattic/wp-calypso/pull/98789/commits/548b4b7d695abda4611b65c519392b6294480ba4